src/bootstrap.js

```diff
@@ -2,9 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-/* globals XPCOMUtils, UAOverrider UAOverrides */
+/* globals XPCOMUtils, Services, UAOverrider UAOverrides */
 
 Components.utils.import("resource://gre/modules/XPCOMUtils.jsm");
+
+const UA_ENABLE_PREF_NAME = "extensions.webcompat.perform_ua_overrides";
```

Comment:
> If so, does it make sense to make a more general pref?

Maybe. Maybe it's best to have a pref per patch type. For now, this feels more natural at the moment. We can rename prefs if we'd like, so...

> is the extensions namespace (prefix??) the right place for this

That's what other Go-Faster extensions are doing (well, except for the e10srollout, they created their own prefix/namespace/whatever (branch, I guess), but as for pdfjs and hello, they sticked to `extensions.*`)